Compiles fails on Archlinux

```
libtool: compile:  g++ -DHAVE_CONFIG_H -I. -I../../src -I.. -iquote ../../src/libmidi -g -O2 -MT libmidi/libmidi_la-Midi.lo -MD -MP -MF libmidi/.deps/libmidi_la-Midi.Tpo -c ../../src/libmidi/Midi.cpp  -fPIC -DPIC -o libmidi/.libs/libmidi_la-Midi.o
../../src/libmidi/Midi.cpp: In member function ‘MidiEventListWithTrackId Midi::Update(microseconds_t)’:
../../src/libmidi/Midi.cpp:408:104: error: no matching function for call to ‘make_pair(size_t&, __gnu_cxx::__alloc_traits<std::allocator<MidiEvent> >::value_type&)’
 ert(aggregated_events.end(), make_pair<size_t, MidiEvent>(i, track_events[j]));
                                                                             ^
In file included from /usr/include/c++/6.2.1/bits/stl_algobase.h:64:0,
                 from /usr/include/c++/6.2.1/bits/char_traits.h:39,
                 from /usr/include/c++/6.2.1/ios:40,
                 from /usr/include/c++/6.2.1/ostream:38,
                 from /usr/include/c++/6.2.1/iostream:39,
                 from ../../src/libmidi/Midi.h:12,
                 from ../../src/libmidi/Midi.cpp:9:
/usr/include/c++/6.2.1/bits/stl_pair.h:425:5: note: candidate: template<class _T1, class _T2> constexpr std::pair<typename std::__decay_and_strip<_Tp>::__type, typename std::__decay_and_strip<_T2>::__type> std::make_pair(_T1&&, _T2&&)
     make_pair(_T1&& __x, _T2&& __y)
     ^~~~~~~~~
/usr/include/c++/6.2.1/bits/stl_pair.h:425:5: note:   template argument deduction/substitution failed:
../../src/libmidi/Midi.cpp:408:104: note:   cannot convert ‘i’ (type ‘size_t {aka long unsigned int}’) to type ‘long unsigned int&&’
 ert(aggregated_events.end(), make_pair<size_t, MidiEvent>(i, track_events[j]));
                                                                             ^
make[2]: *** [Makefile:547: libmidi/libmidi_la-Midi.lo] Error 1
```

--- a/src/libmidi/Midi.cpp
+++ b/src/libmidi/Midi.cpp
@@ -403,9 +403,9 @@ MidiEventListWithTrackId Midi::Update(microseconds_t delta_microseconds) {
     MidiEventList track_events = m_tracks[i].Update(delta_microseconds);
 
     const size_t event_count = track_events.size();
-    // Collect evants to be passed to a screen keyboard
+    // Collect events to be passed to a screen keyboard
     for (size_t j = 0; j < event_count; ++j) {
-      aggregated_events.insert(aggregated_events.end(), make_pair<size_t, MidiEvent>(i, track_events[j]));
+      aggregated_events.insert(aggregated_events.end(), make_pair(i, track_events[j]));
     }
   }
 

